components/isceobj/RtcProc/runPreprocessor.py

```diff
@@ -72,10 +72,10 @@ def runPreprocessor(self):
     catalog.printToLog(logger, "runPreprocessor")
     self._grd.procDoc.addAllFromCatalog(catalog)
 
-def extract_slc(sensor, slantRange=False):
+def extract_slc(sensor, slantRange=False, removeNoise=False):
 #    sensor.configure()
     sensor.parse()
-    sensor.extractImage()
+    sensor.extractImage(removeNoise=removeNoise)
```

Comment:
I think you cannot expect to have this ```removeNoise``` argument for every sensor (even though they are currently provided for all available GRD sensors). You may want to check it first by using ```inspect.getargspec()``` or just by calling it within a try/except.

Reply:
Right, will do.

Reply:
Done in https://github.com/isce-framework/isce2/pull/192/commits/742d3b48c7bfbd615f5d8c70f8489a3e63b6fe79

Reply:
Thank you, @vincentschut !